source/releasenotes/2025-05-07-terminus-4-0-0.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Terminus 4.0.0 release now available"
+published_date: "2025-05-07"
+categories: [tools-apis]
+---
+
+Terminus [4.0.0](https://github.com/pantheon-systems/terminus/releases/tag/4.0.0) is now available.
+This release adds support for PHP 8.4 and removes support for PHP versions 8.1 and below.
+Terminus 4 is compatible with PHP 8.2 and 8.3.
+See [Supported Terminus and PHP versions](/terminus/supported-terminus).
+The PHP version refers to the local machine on which Terminus itself runs (e.g. developers' own computers or containers for continuous integration). This is distinct from PHP versions that run websites on Pantheon's infrastructure (which still include support for PHP 7.x).
+
+For more information about this release, visit the [GitHub release page](https://github.com/pantheon-systems/terminus/releases/tag/4.0.0).
+
+## Terminus 3 to be supported through May 7th, 2026
+
+It is Pantheon's policy to support a version of Terminus for one year from the release of the subsequent version. With the release of Terminus 4 today, Pantheon will continue to support Terminus 3 (bug fixes, but not new feature development) through May 7th, 2026.
```

Comment:
This doesn't quite describe the policy. We allow folks to *RUN* Terminus 3.6.2 until May 07, 2026. This doesn't necessarily mean that we will also fix bugs in this release, though, as that is a separate policy. Supporting bug fixes in Terminus 3 is at the discretion of @scottbuscemi .

Reply:
I've removed this section. It doesn't make sense to include if the EoL date is not representative of support.